src/runtime/components/navigation/Stepper.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <ol :class="[ui.base]">
+    <li
+      v-for="(item, index) of items"
+      :key="index"
+      ref="itemRefs"
+      :class="[ui.list.base]"
+    >
+      <div
+        v-if="displayDivider"
+        :class="[index > 0 ? 'visible' : 'invisible', ui.list.step.divider]"
+      />
+      <UButton
+        color="white"
```

Comment:
To customize how the button look, we might need to do something similar to the `Accordion` component: https://github.com/nuxt/ui/blob/dev/src/runtime/components/elements/Accordion.vue#L6, https://github.com/nuxt/ui/blob/dev/src/runtime/app.config.ts#L307.

Reply:
So use a ULink instead of UButton?